plugins/inputs/kinesis_consumer/kinesis_consumer.go

```diff
@@ -89,6 +91,25 @@ func (k *KinesisConsumer) SetParser(parser telegraf.Parser) {
 	k.parser = parser
 }
 
+type TelegrafLoggerWrapper struct {
+	telegraf.Logger
+}
+
+func (t *TelegrafLoggerWrapper) Log(args ...interface{}) {
+	t.Debug(args...)
```

Comment:
I would really rather use the new `Trace` log-level for this to avoid spamming the log for people enabling debugging.